### PR TITLE
Update VOTO cdl with instruments

### PR DESCRIPTION
**Pull request by callumrollo**
# Type of PR

- [x] Typo without possible change of interpretation of the related text.

### CODE REVIEW

**Comment by callumrollo** [comment]
This is now ready for review. It passes Gui's OG1.0 checker

**Review by glidermann — APPROVED** [review]

**Review by castelao — APPROVED** [review]
It looks good to me, but I missed the latest discussions. It might be worth waiting for another pair of eyes here.

I'm curious. did you find the data types of the attributes through the checker? That is a common mistake and sometimes not easy to notice.

I suggest adding a comment that this is a subset for [demonstration purposes](https://github.com/OceanGlidersCommunity/OG-format-user-manual/blob/fd2bbaa9fd143a53f41aaef05ae731c000c3009b/src/sp041_20191205T1757.cdl#L219). That might avoid any confusion if somehow this dataset is placed on the side of the original one.

Well done!

**Comment by callumrollo** [comment]
Thanks for the review Gui! The checker found several dtype issues that I hadn't caught, particularly making sure that the dtype of min and max values is the same as that of the data itself. It's been very useful.

I've added the truncated datasets comment.

I'll hold off on merging this until after the meeting on Wednesday to give others a chance to review

**Pull request merged**
